### PR TITLE
Moving relationships out of "Reputation"

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Description
Moving the documentation for relationships to a more correct location.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/indicators/relationships.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+EntityRelationship(
+   name='contains',
+   entity_a='1.1.1.1',
+   entity_a_type='IP',
+   entity_b='2.2.2.2',
+   entity_b_type='IP',
+   source_reliability='B - Usually reliable',
+   brand='My Integration ID')
+```
+   - When setting the name of the relationship, make sure to choose a value that appear in the the predefined list of [relationships](https://xsoar.pan.dev/docs/reference/api/common-server-python#relationships.)
```
Comment (ShirleyDenkberg) [inline docs/indicators/relationships.md:32]:
```suggestion
   - When setting the name of the relationship, make sure to choose a value that appears in the the predefined list of [relationships](https://xsoar.pan.dev/docs/reference/api/common-server-python#relationships.)
```

File: docs/indicators/relationships.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+   - For more information about creating a relationship entity, visit the [EntityRelationship](https://xsoar.pan.dev/docs/reference/api/common-server-python#entityrelationship).
```
Comment (ShirleyDenkberg) [inline docs/indicators/relationships.md:34]:
```suggestion
   - For more information about creating a relationship entity, visit [EntityRelationship](https://xsoar.pan.dev/docs/reference/api/common-server-python#entityrelationship).
```

**Comment by ShirleyDenkberg** [comment]
@Ni-Knight Doc review completed.

**Comment by Ni-Knight** [comment]
@ShirleyDenkberg - committed suggestions (Not my changes, just moved text location) :)

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**